source/transactions/tests/unified/pin-mongos.yml

```diff
@@ -572,6 +573,7 @@ tests:
                 - isMaster
                 - hello
               closeConnection: true
+              appName: *appName
```

Comment:
I think these transaction test changes look good. I believe couldn't use this feature originally because of this server bug: SERVER-49336. Can you confirm that these tests pass on current 4.2 and up servers?

Reply:
Yes, the tests are green.

Reply:
There are 2 possible fixes for now: use appName with additional connection for failPoints OR simply wait for all failPoints to be processed and then remove fialPoint (the second approach makes us waste around of 3-4 seconds waiting)